curator/patterns/post-hero.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Single Post Hero
+ * Slug: curator/post-hero
+ * Inserter: no
+ */
+
+?>
+
+<?php if ( has_post_thumbnail() ): ?>
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":50,"overlayColor":"foreground","minHeight":100,"minHeightUnit":"vh"} -->
+<div class="wp-block-cover" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:post-terms {"term":"post_tag","style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"textColor":"background","className":"curator-post-tags","fontSize":"medium"} /-->
+
+<!-- wp:post-date {"textColor":"background","fontSize":"medium"} /--></div>
```

Comment:
There is a pattern created for the terms and date [here](https://github.com/Automattic/themes/pull/5994/files#diff-73a0eafe38a2cdc775a3a9f5898b911ea762d3d1e36f1f2d3c32854a55e05109). Please use it once the PR gets merged.